app/move/controllers/create/assessment.test.js

```diff
@@ -1,52 +1,217 @@
 const FormController = require('hmpo-form-wizard').Controller
+const referenceDataService = require('../../../../common/services/reference-data')
+const createFields = require('../../../move/fields/create')
 
 const Controller = require('./assessment')
-const fieldHelpers = require('../../../../common/helpers/field')
-
 const controller = new Controller({ route: '/' })
 
-const assessmentQuestionFieldsMock = {
-  risk: {
-    name: 'risk',
-    items: [
-      {
-        value: 'd3a50d7a-6cf4-4eeb-a013-1ff8c5c47cc1',
-        text: 'Violent',
-        key: 'violent',
-        conditional: 'risk__violent',
-      },
-    ],
-  },
-  risk__violent: {
-    skip: true,
-    rows: 3,
-    component: 'govukTextarea',
-    classes: 'govuk-input--width-20',
-    label: {
-      text: 'fields::assessment_comment.required',
-      classes: 'govuk-label--s',
-    },
-    validate: 'required',
-    dependent: {
-      field: 'risk',
-      value: 'd3a50d7a-6cf4-4eeb-a013-1ff8c5c47cc1',
-    },
-  },
-  first_names: {
-    name: 'first_names',
-  },
-}
-
 describe('Move controllers', function() {
   describe('Assessment controller', function() {
+    let nextSpy
+    let refDataService
+    let req
+    describe('#configure() 2', function() {
```

Comment:
Perhaps merge the 2 describe blocks testing configure() and have separate `context`s for the different setup data?